crates/bevy_ui/src/ui_node.rs

```diff
@@ -1832,14 +1839,43 @@ pub struct UiImage {
     pub flip_y: bool,
 }
 
+impl Default for UiImage {
+    /// A solid square, with a transparent white color.
+    fn default() -> Self {
+        UiImage {
+            // This needs to be transparent by default, to avoid covering the background color
+            // This should be white because the tint is multiplied with the image,
+            // so if you set an actual image with default tint you'd want its original colors
+            color: Color::from(LinearRgba::new(1.0, 1.0, 1.0, 0.0)),
```

Comment:
Ideally the default `color` should be `Color::WHITE`, and the default `texture` should (somehow) be a transparent square. Otherwise `UiImage { texture: my_image, ..default() }` will be invisible. Using `UiImage::new(my_image)` instead does work, at least.

Reply:
If this behavior is considered acceptable, I'm fine with approving this PR, since everything else looks good to me.

Reply:
Unfortunately changing the default Image to be transparent will be very annoying for sprites, since then spawning a sprite will seemingly nothing by default.

It's also badly breaking in a much more complex way. I don't like this either, but I don't see a better way forward.

Reply:
Could there be like `Handle::<Image>::transparent()`?

Reply:
This would require us to somehow register this by default during the asset initialization of `Image`. I'm not sure how to do that, and I'm worried about the increased scope.